### PR TITLE
feat: [EAM-3668] Non conformity observation

### PR DESCRIPTION
**Pull request by ricardo-j-almeida**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -424,6 +424,10 @@
                 <wsdlUrl>${infor.wsdls}/Asset/MP3400_GetNonconformity_001.wsdl</wsdlUrl>
                 <wsdlUrl>${infor.wsdls}/Material/MP1287_Store2StoreTransfer_001.wsdl</wsdlUrl>
                 <wsdlUrl>${infor.wsdls}/Work/MP7927_ProcessContractTemplate_001.wsdl</wsdlUrl>
+                <wsdlUrl>${infor.wsdls}/Work/MP3442_AddNonconformityObservation_001.wsdl</wsdlUrl>
+                <wsdlUrl>${infor.wsdls}/Work/MP3443_SyncNonconformityObservation_001.wsdl</wsdlUrl>
+                <wsdlUrl>${infor.wsdls}/Work/MP3444_DeleteNonconformityObservation_001.wsdl</wsdlUrl>
+                <wsdlUrl>${infor.wsdls}/Work/MP3445_GetNonconformityObservation_001.wsdl</wsdlUrl>
```
Comment (ricardo-j-almeida) [inline pom.xml]: You are missing the merge of the files that is configured below

Comment (ricardo-j-almeida) [reply pom.xml]: See 5467ea24ef66c4ce1fe4bd259b521b34d2474694 for an example

Comment (DevauxJ) [reply pom.xml]: Done

Comment (ricardo-j-almeida) [inline pom.xml]: Change the folder structure to Asset

File: pom.xml [hunk 2]
```diff
@@ -424,6 +424,10 @@
                 <wsdlUrl>${infor.wsdls}/Asset/MP3400_GetNonconformity_001.wsdl</wsdlUrl>
                 <wsdlUrl>${infor.wsdls}/Material/MP1287_Store2StoreTransfer_001.wsdl</wsdlUrl>
                 <wsdlUrl>${infor.wsdls}/Work/MP7927_ProcessContractTemplate_001.wsdl</wsdlUrl>
+                <wsdlUrl>${infor.wsdls}/Work/MP3442_AddNonconformityObservation_001.wsdl</wsdlUrl>
```
Comment (ricardo-j-almeida) [inline pom.xml]: Add:
MP3441_GetNonconformityObservationDefault_001.wsdl

**Pull request merged**